config/webhook/manifests.yaml

```diff
@@ -59,3 +59,91 @@ webhooks:
         scope: '*'
     sideEffects: None
     timeoutSeconds: 15
+
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: mutating-webhook.kyma-project.io
+webhooks:
+  - name: mutating.v1alpha1.metricpipelines.telemetry.kyma-project.io
```

Comment:
Does the webhook name also has to follow this specific pattern for the kubebuilder logic to work?
If not, then maybe we can rename it to `mutating-metricpipelines-v1alpha1.kyma-project.io`? to make it have a similar pattern to the names for the validating webhooks. Also, we can do the same to the other names of the mutating webhooks.

Reply:
No, this name not generated. The names updated to be consistent with others